addons/units/CfgWeapons.hpp

```diff
@@ -192,7 +192,58 @@ class CfgWeapons {
             uniformClass = QCLASS(Unit_Combat_LS_WS_WP_WB);
         };
     };
+
     // Plaid
+    class CLASS(Uniform_Combat_LS_CBPS_BP_BB): CLASS(Uniform_Base) { // BPS = Buffalo Plaid Shirt
+        scope = 2;
+        author = "GilleeDoo, Tyrone";
+        displayName = CSTRING(Uniform_Combat_LS_CBPS_BP_BB);
+        picture = QPATHOF(UI\uniform_combat_ls_cbps_bp_bb_ca.paa);
```

Comment:
Also, uniform pictures have no boots, so their names are without boots:
```suggestion
        picture = QPATHOF(UI\uniform_combat_ls_cbps_bp_ca.paa);
```